### PR TITLE
Fixes Bug with TabbedPage pushed onto Shell

### LINKED ISSUE #6384: [Bug] [iOS] content page in tabbed page not showing inside shell tab
### Description
Content inside tabbed page showing black/blank page when place a tabbed page inside shell tabbar or after Shell is navigate to another Tabbed page. 
### Steps to Reproduce

1. Create a shell with bottom tabbar items.
2. Place a tabbed page for one of the tab content or navigate to a tabbed page from shell.

### Expected Behavior
The tabbed page's content is expected to be displayed.

### Actual Behavior
The tabbed page's content display as black content.

### Basic Information

- Version with issue: 4.0.0.425677
- Last known good version:
- IDE: Visual Studio Community 2017 (15.9.6) 
- Platform Target Frameworks: 
  - iOS: 12.2  
- Nuget Packages: Basic Nugets from a fresh project

### Screenshots

Screenshots from simulator
2019-06-03_02-33-17-PM
2019-06-03_02-33-30-PM

### Reproduction Link

[TabbedPageBlankContent.zip](https://github.com/xamarin/Xamarin.Forms/files/3246157/TabbedPageBlankContent.zip)

### PR DESCRIPTION
**Pull request by leoMehlig**
### Description of Change ###

Fixes an issue when pushing a TabbedPage onto a Shell. 

On iOS the `TabbedRenderer` did not propagate size updates to the children, which resulting in them being hidden.

On Android the `PageRenderer` did not set the default background for the children of the `TabbedPage` and since the `TabbedPage` does not have a background the shell was still visible.

### Issues Resolved ### 
- fixes #6384

### API Changes ###
 
 None

### Platforms Affected ### 
- iOS
- Android

### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 
android-bug
android-fix
iphone-bug
iphone-fixed

### Testing Procedure ###
The `Github6384` test page shows this fix.
*Note: The UI Test does only catch the bug on iOS and not on Android. I wasn't able to check the background color of a nativ view in the UI Test.*

### PR Checklist ###

- [ ] Targets the correct branch
- [ ] Tests are passing (or failures are unrelated)

### CODE REVIEW

File: Xamarin.Forms.Platform.Android/Renderers/PageRenderer.cs [hunk 1]
```diff
@@ -137,7 +137,11 @@ void UpdateBackground(bool setBkndColorEvenWhenItsDefault)
 				{
 					Color bkgndColor = page.BackgroundColor;
 					bool isDefaultBkgndColor = bkgndColor.IsDefault;
-					if (page.Parent is BaseShellItem && isDefaultBkgndColor)
+
+					// A TabbedPage has no background. See Github6384.
+					bool isInShell = page.Parent is BaseShellItem
+					|| (page.Parent is TabbedPage && page.Parent.Parent is BaseShellItem);
```
Comment (rmarinho) [inline Xamarin.Forms.Platform.Android/Renderers/PageRenderer.cs]:
```suggestion
					|| (page.Parent is TabbedPage && page.Parent?.Parent is BaseShellItem);
```

**Comment by leoMehlig** [comment]
Since I'm no longer working at @jedox I can't push to the branch. But I will inform my former colleges.

File: Xamarin.Forms.Platform.Android/Renderers/PageRenderer.cs (hunk 1 above)
Comment (hadi-zamani) [reply Xamarin.Forms.Platform.Android/Renderers/PageRenderer.cs]: I made the requested change.

Comment (hadi-zamani) [reply Xamarin.Forms.Platform.Android/Renderers/PageRenderer.cs]: I made the suggested change.

**Comment by hadi-zamani** [comment]
@samhouts The conflicts in Xamarin.Forms.Controls.Issues.Shared.projitems should be resolved by me?

**Pull request merged**